clientgui/WizardAttach.cpp

```diff
@@ -580,6 +579,9 @@ void CWizardAttach::_ProcessCancelEvent( wxWizardExEvent& event ) {
     GetNextButton()->Enable(HasNextPage(page));
     GetBackButton()->Enable(HasPrevPage(page));
 
+#if 0   // These appear to no longer be used
```

Comment:
I don't like dead code. So I'd suggest to remove it until you have strong arguments why we should kept it.